tools/depends/target/python3/Makefile

```diff
@@ -10,28 +10,43 @@ ARCHIVE=$(SOURCE).tar.xz
 BASE_URL=https://www.python.org/ftp/python/$(VERSION)
 
 ifeq ($(findstring apple-darwin, $(HOST)), apple-darwin)
-  HOSTPLATFORM=_PYTHON_HOST_PLATFORM="darwin"
-  LINK_ICONV=-framework CoreFoundation -liconv
+  LDFLAGS+= -framework Foundation
+  ifeq ($(OS), darwin_embedded)
+    EXTRA_CONFIGURE=ac_cv_func_wait3=no ac_cv_func_wait4=no ac_cv_func_waitpid=no \
+                    ac_cv_func_execv=no ac_cv_func_fexecv=no ac_cv_func_getentropy=no \
+                    ac_cv_func_setpriority=no ac_cv_func_sendfile=no ac_cv_header_sched_h=no \
+                    ac_cv_func_posix_spawn=no ac_cv_func_posix_spawnp=no \
+                    ac_cv_func_forkpty=no ac_cv_lib_util_forkpty=no \
+                    ac_cv_func_getgroups=no
+  endif
 endif
 
-ifeq ($(OS), darwin_embedded)
-  EXTRA_CONFIGURE=ac_cv_func_wait3=no ac_cv_func_wait4=no ac_cv_func_waitpid=no \
-                  ac_cv_func_execv=no ac_cv_func_fexecv=no ac_cv_func_getentropy=no \
-                  ac_cv_func_setpriority=no ac_cv_func_sendfile=no ac_cv_header_sched_h=no \
-                  ac_cv_func_posix_spawn=no ac_cv_func_posix_spawnp=no \
-                  ac_cv_func_forkpty=no ac_cv_lib_util_forkpty=no \
-                  ac_cv_func_getgroups=no
+ifeq ($(CROSS_COMPILING),yes)
+	CROSSBUILD=--host=$(HOST) \
+			   --build=$(BUILD)
 endif
 
-# configuration settings
-CONFIGURE=./configure --prefix=$(PREFIX) \
-  --disable-shared \
-  --without-ensurepip \
-  --disable-framework --with-system-expat --with-system-ffi --without-pymalloc --enable-ipv6 \
-  $(EXTRA_CONFIGURE)
+COMPILE_HOST_PATH := $(PATH)
```

Comment:
this extra variable seems uneeded?

Reply:
I'll have to look at it again. Was a point at some time, but may have been many revisions ago, so might not be relevant currently

Reply:
Removed as unneccessary.